mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -39,14 +47,22 @@ $userNameFontSize: 16px;
   gap: $gapSmall;
 }
 
-.socialMediaIcons {
+.socialMediaText {
+  font-weight: 500;
+}
+
+.iconWrapper {
+  display: flex;
+  width: $iconSize;
+  height: $iconSize;
   align-items: center;
+  justify-content: center;
+  border-radius: $tertiaryBorderRadius;
+  background-color: var(--iconBgColor);
 }
 
 .socialMediaIcon {
-  border: 2px solid var(--secondaryStrokeColor);
-  border-radius: $tertiaryBorderRadius;
+  display: block;
   background-color: var(--iconBgColor);
-  fill: var(--quaternaryBgColor);
-  stroke: none;
+  stroke: var(--quaternaryBgColor);
```

Comment:
stroke is not Background

Reply:
For stroke we don't have a variable that will be dark and won't be changes in dark/light mode. I'm waiting for a response from the designers

Reply:
Changed to a new variable var(--tertiaryIconColor), which also added to global variables